cypress/fixtures/product/simple_product.json

```diff
@@ -1,23 +1,14 @@
 {
-    "name": "ProductName",
-    "description": "TestProduct1",
-    "value": "TestProduct1",
-    "m_product_category": "06_Kartoffeln",
-    "c_uom": "Each",
-    "de_DE__c_uom": "Stück",
-    "isStocked": true,
-    "isPurchased": true,
-    "isSold": true,
-    "isDiverse": false,
-    "productType": "Item",
-    "de_DE__productType": "Artikel",
-    "prices": [
-      {
-        "priceStd": 2.00,
-        "m_pricelist_version": "Testpreise Kunden (Deutschland)_Germany - Deutschland_EUR_2015-01-01",
-        "de_DE__m_pricelist_version": "Testpreise Kunden (Deutschland)_Deutschland_EUR_2015-01-01",
-        "c_taxcategory": "Regular Tax Rate 19% (Germany)",
-        "de_DE__c_taxcategory": "Normaler Steuersatz 19% (Deutschland)"
-      }
-    ]
-}
\ No newline at end of file
+  "name": "ProductName",
+  "description": "TestProduct1",
+  "value": "TestProduct1",
+  "m_product_category": "06_Kartoffeln",
+  "c_uom": "Each",
+  "de_DE__c_uom": "Stück",
+  "isStocked": true,
+  "isPurchased": true,
+  "isSold": true,
+  "isDiverse": false,
+  "productType": "Item",
+  "de_DE__productType": "Artikel"
```

Comment:
pls make sure to port those recent trls like `de_DE__c_taxcategory`

Reply:
Fixed in ProductPrice fixture